src/util.rs

```diff
@@ -197,7 +190,7 @@ where
     // Finish writing stdin before waiting, because waiting drops stdin.
 
     if let Some(stdin) = stdin {
-        stdin.await;
+        stdin.await.unwrap();
```

Comment:
`unwrap`s terminate on error, but most of the time, we want to warn. So either an `let _ = ` (i.e. in this case) or a `warn!` should be fine.